contracts/modules/AaveWithdrawModule.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "interfaces/gnosis/IGnosisSafe.sol";
+import {DataTypes, IAaveV3Pool} from "interfaces/aave/IAaveV3Pool.sol";
+import "interfaces/aave/IAToken.sol";
+
+import {BaseModule} from "../BaseModule.sol";
+
+contract AaveWithdrawModule is BaseModule {
+    ////////////////////////////////////////////////////////////////////////////
+    // CONSTANTS
+    ////////////////////////////////////////////////////////////////////////////
+
+    IAaveV3Pool AAVE_POOL = IAaveV3Pool(0x7b5C526B7F8dfdff278b4a3e045083FBA4028790);
+
+    ////////////////////////////////////////////////////////////////////////////
+    // EVENTS
+    ////////////////////////////////////////////////////////////////////////////
+
+    event EmergencyWithdraw(address asset, uint256 amount, address signer, uint256 timestamp);
+
+    function withdraw(IGnosisSafe safe, address collateral) external isSigner(safe) {
+        if (!safe.isModuleEnabled(address(this))) revert ModuleMisconfigured();
+
+        DataTypes.ReserveData memory reserveData = AAVE_POOL.getReserveData(collateral);
```

Comment:
sanity check on `collateral` first?

Reply:
this is interesting, need to look later on how to check it

Reply:
check added ba6b789981c1235551dd73837d34b73120442af1

Reply:
nice solution!